pages/common/go-test.md

```diff
@@ -0,0 +1,24 @@
+# go test
+
+> Tests Go packages (files have to end with `_test.go`).
+> More information: <https://golang.org/cmd/go/#hdr-Testing_flags>.
+
+- Test the package found in the current directory:
+
+`go test`
+
+- [v]erbosely test the package in the current directory:
+
+`go test -v`
+
+- [v]erbosely test the packages in the current directory and all subdirectories:
+
+`go test -v ./...`
+
+- [v]erbosely test the package in the current directory and run all benchmarks:
+
+`go test -v -bench .`
+
+- [v]erbosely test the package in the current directory and run all benchmarks for 50 seconds:
```

Comment:
Since `-v` is in all of the examples, why not just omit explaining it?  I'd like to keep using `[` as the first character at a minimum.
```suggestion
- Test the packages in the current directory and all subdirectories:

`go test -v ./...`

- Test the package in the current directory and run all benchmarks:

`go test -v -bench .`

- Test the package in the current directory and run all benchmarks for 50 seconds:
```